Enhancement/updates

clente opened the pull request:
These changes could potentially break current automated workflows. Make sure this doesn't happen before merging.

* List authors and require updated version of `CursoRopen`
* Update (and simplify) GitHub Actions
* Simplify [README.Rmd](README.Rmd) with new functions from `CursoRopen` (see https://github.com/curso-r/CursoRopen/pull/2)

clente committed:
Use updated version of CursoRopen

clente committed:
Use updated version of r-lib/actions and don't share PAT

clente committed:
Remove r-depends

clente committed:
Use updated version of r-lib/actions

clente committed:
Update inst/ files

clente committed:
Use newer version of CursoRopen

clente committed:
Update DESCRIPTION

clente committed:
Increment version number to 0.0.0.9001

beatrizmilz commented:
Oi caio! eu só vi esse PR agora. preciso dar uma verificada nos conflitos pois apaguei alguns arquivos!

beatrizmilz committed:
merge

Merge branch 'main' into enhancement/updates

# Conflicts:
#	README.Rmd
#	README.md
#	inst/atualizar_pagina.R
#	inst/atualizar_pagina_do_curso.yaml
